doc/devdocs/modules/launcher/plugins/history.md

```diff
@@ -6,7 +6,7 @@ The History Plugin allows users to search or display results they have used (sel
 The plugin uses data that was already being captured which is, what results were clicked, and how many times. We do add a little more data to this set now.
 When this plugin is queried, it creates results based on this previously selected results data.
 
-In order to make sure selected results in the history are still valid, we re-query the plugin the relevant plug using the PluginManager. If there are no results,
+In order to make sure selected results in the history are still valid, we re-query the relevant plugin using the PluginManager. If there are no results,
```

Comment:
This was quite odd. I looked at the history of the line and gave up. This is my best guess.

Reply:
This sounds right to me. Thanks!